### DISCUSSION TITLE
Final Days of Hacktoberfest'23: Get your PR's Submitted! :octocat:

### DISCUSSION THREAD

**Discussion by kehoecj** [Boeing/config-file-validator#80] [Announcements]
We're entering the final days of Hacktoberfest 2023 🎃 👻 🍂 ! This has been such an incredible experience for us and we are so grateful for everyone who has contributed to the config-file-validator. We have reviewed and merged 14 PR's so far in October.

If you've been assigned an issue please get the PR submitted as soon as possible so there is time for us to review it and for you to make any recommended changes. PR's submitted on October 31 will very likely not be merged in time to count. If you have any questions please reach out in the issue!

**Comment by zhenrong-wang** [comment]
This is really a powerful tool for guys who have to deal with different types of configuration files.
